problems/4.arrays.js

```diff
@@ -1,9 +1,10 @@
-/** Averages the first and last value of an array.  Return null if the array has fewer than 2 elements.
+/** Averages the first and last value of an array.  
+ * Return null if the array has fewer than 2 elements.
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+const firstAndLastAverage = (arr) => {
+  return arr.shift() + arr.pop(); 
```

Comment:
This is close. Average would also be to divide by 2. 
Be aware that shift and pop will mutate the array. Safest for this problem would be to just key into them.